README.md

```diff
@@ -63,139 +68,219 @@ Use `#pragma mark -` to categorize methods in functional groupings and protocol/
 ```
 
 ## Spacing
+
+### Indentation
+* **Indent using 4 spaces. Never indent with tabs.** Be sure to set this preference in Xcode.
+* Method braces and other braces (`if`/`else`/`switch`/`while` etc.) always open on the same line as the statement but close on a new line.
+* One whitespace before the parenthesis, no space inside parenthesis
+
+**For example:**
+
+```objc
+if (user.isHappy) {
+    // Do something
+}
+else {
+    // Do something else
+}
+```
+### Line Break and Whitespace
+* There should be exactly one blank line between methods to aid in visual clarity and organization.
+* Whitespace within methods should be used to separate functionality (though often this can indicate an opportunity to split the method into several, smaller methods). In methods with long or verbose names, a single line of whitespace may be used to provide visual separation before the method’s body.
+
+**For example**
+
+```objc
+- (void)initWithUser:(User *)newUser firstname:(NSString *)firstname lastname:(NSString *)lastname;
+```
+
+* `@synthesize` and `@dynamic` should each be declared on new lines in the implementation.
+
+### Alignmnent
+* Align "=" signs within the same code block
+
+
 ## Naming
-## Properties
-## Methods
-## Variables
-## Constants
-## Enumeration Types
-## Bitmasks
-## Singletons
-## Comments
 
+Apple naming conventions should be adhered to wherever possible, especially those related to [memory management rules](https://developer.apple.com/library/mac/#documentation/Cocoa/Conceptual/MemoryMgmt/Articles/MemoryMgmt.html) ([NARC](http://stackoverflow.com/a/2865194/340508)).
 
-<!--## Dot Notation Syntax
+Long, descriptive method and variable names are good.
```

Comment:
I think that this is something where there are better guidelines for the length of names. The guideline that I’ve always used is that the length of a variable name should roughly correspond to how important it is and how long it lives.

Thus, `i` as an index counter in a loop is *perfect* because it has no life outside of a loop. A `Point` class would obviously have `x`, `y` (and `z`  if it’s in 3D space) because they are perfectly descriptive. A method name or variable name that is *excessively* long, however, benefits no one.